File: boat-test/src/test/java/test/xyz/srclab/test/mark/TestMarkerTest.java
package test.xyz.srclab.test.mark;

import org.testng.annotations.Test;
import xyz.srclab.test.mark.TestMarker;

/**
 * @author sunqian
 */
public class TestMarkerTest {

    @Test
    public void testTestMarker() {
        TestMarker testMarker = TestMarker.newTestMarker();
        testMarker.mark("hello");
        testMarker.mark("world", "!");
        System.out.println(testMarker);
    }
}

File: boat-test/src/test/kotlin/test/xyz/srclab/test/mark/TestMarkerTest.kt
package test.xyz.srclab.test.mark

import xyz.srclab.test.mark.TestMarker
import kotlin.test.Test

object TestMarkerTestKt {

    @Test
    fun testTestMarker() {
        val testMarker = TestMarker.newTestMarker()
        testMarker.mark("hello")
        testMarker.mark("world", "!")
        println(testMarker)
    }
}